src/common/resources/poLine.js

```diff
@@ -0,0 +1,15 @@
+import { get } from 'lodash';
+
+import { baseManifest } from '@folio/stripes-acq-components';
+
+import { PO_LINES_API } from '../constants';
+
+// eslint-disable-next-line import/prefer-default-export
+export const poLineResource = {
+  ...baseManifest,
+  path: (queryParams, pathComponents, resourceData, logger, props) => {
+    const poLineId = get(props, ['resources', 'invoiceLine', 'records', 0, 'poLineId']);
```

Comment:
IMHO, it's not a common resource at all, because it's tightly coupled to other resources. Such functions is better to write directly in manifests, since related resource names might differ.

Reply:
also such conception isn't bug-proof from the case when clicking on another invoice line, that doesn't have `poLineId` - path will be null, but resource still contains data for previous poLineId. As a solution we decided to load resources in a manual way on useEffect and to store things in state.